micrometer-support/src/main/java/io/javaoperatorsdk/operator/monitoring/micrometer/MicrometerMetrics.java

```diff
@@ -153,45 +155,53 @@ private static String getScope(ResourceID resourceID) {
 
   @Override
   public void receivedEvent(Event event, Map<String, Object> metadata) {
-    final String[] tags;
-    if (event instanceof ResourceEvent) {
-      tags = new String[] {"event", event.getClass().getSimpleName(), "action",
-          ((ResourceEvent) event).getAction().toString()};
-    } else {
-      tags = new String[] {"event", event.getClass().getSimpleName()};
+    if (collectPerResourceMetrics) {
```

Comment:
we could still have a counter per resource kind ? Or does each event trigger logic that increment another counter ?